install

```diff
@@ -0,0 +1,593 @@
+    fi
+fi
+
+if [ "$icecast" = "f" -a ${_i} -eq 1 ]; then
+    echo -e "Install default Airtime Icecast configuration? (Y/n): \c"
+    read IN
+    if [ "$IN" = "y" -o "$IN" = "Y" ]; then
+        icecast="t"
+    fi
+fi
+
+if [ "$icecast" = "t" ]; then
+    loud "\n-----------------------------------------------------"
+    loud "               * Installing Icecast *                "
+    loud "-----------------------------------------------------"
+    
+    loudCmd "DEBIAN_FRONTEND=noninteractive apt-get -y --force-yes install icecast2"
+
+    verbose "\n * Enabling Icecast 2..."
+    sed -i 's/ENABLE=false/ENABLE=true/g' /etc/default/icecast2
+    set +e
+    service icecast2 start
+    set -e
+    verbose "...Done"
+fi
+
+loud "\n-----------------------------------------------------"
+loud "          * Installing Airtime Services *            "
+loud "-----------------------------------------------------"
+
+verbose "\n * Installing necessary binaries..."
+loudCmd "apt-get -y --force-yes install liquidsoap python python-pip"
+loudCmd "pip install setuptools"
+verbose "...Done"
+
+verbose "\n * Creating /usr/lib/airtime..."
+mkdir -p /usr/lib/airtime
+verbose "...Done"
+
+verbose "\n * Creating /run/airtime..."
+mkdir -p /run/airtime
+chmod 755 /run/airtime
+chown -R ${web_user}:${web_user} /run/airtime
+verbose "...Done"
+
+verbose "\n * Copying logging files..."
+cp -R ${AIRTIMEROOT}/python_apps/std_err_override /usr/lib/airtime/
+verbose "...Done"
+
+verbose "\n * Copying API client files..."
+cp -R ${AIRTIMEROOT}/python_apps/api_clients /usr/lib/airtime/
+verbose "...Done"
+
+verbose "\n * Copying media-monitor files..."
+cp -R ${AIRTIMEROOT}/python_apps/media-monitor/media-monitor /usr/lib/airtime/
+cp -R ${AIRTIMEROOT}/python_apps/media-monitor/media-monitor2 /usr/lib/airtime/media-monitor/mm2
+verbose "...Done"
+
+verbose "\n * Installing media-monitor..."
+python ${AIRTIMEROOT}/python_apps/media-monitor/setup.py install
+verbose "...Done"
+
+verbose "\n * Copying pypo files..."
+mkdir -p /usr/lib/airtime/pypo
+cp -R ${AIRTIMEROOT}/python_apps/pypo/pypo /usr/lib/airtime/pypo/bin/
+verbose "...Done"
+
+verbose "\n * Installing pypo..."
+python ${AIRTIMEROOT}/python_apps/pypo/setup.py install
+verbose "...Done"
+
+verbose "\n * Creating liquidsoap symlink..."
+ln -sf /usr/bin/liquidsoap /usr/bin/airtime-liquidsoap
+verbose "...Done"
+
+for i in /etc/init/airtime*; do
+    chmod 644 $i
+    sed -i "s/WEB_USER/${web_user}/g" $i
+done
+
+initctl reload-configuration
+
+if [ ! -d /var/log/airtime ]; then
+    loud "\n-----------------------------------------------------"
+    loud "              * Installing Log Files *               "
+    loud "-----------------------------------------------------"
+
+    verbose "\n * Creating /var/log/airtime..."
+    mkdir -p /var/log/airtime
+    mkdir -p /var/log/airtime/media-monitor
+    mkdir -p /var/log/airtime/pypo
+    mkdir -p /var/log/airtime/pypo-liquidsoap
```

Comment:
Each component's individual installer should create any directories it needs. 
eg. The setuptools-based installer for pypo should create /var/log/airtime/pypo and /var/log/airtime/pypo-liquidsoap.